src/cmds/blockdev.c

```diff
@@ -0,0 +1,75 @@
+/**
+ * @file
+ *
+ * @date 20.02.23
+ * @author Adityar Rayala
+ */
+
+#include<stdio.h>
+#include<string.h>
+#include<assert.h>
+#include <drivers/block_dev.h>
+
+static void print_usage(void) {
+    printf("Usage:\nblockdev command devices\n");
+    printf("blockdev --report [devices]\n");
+    printf("Options:\n");
+    printf(" %-15s print report for specified (or all) devices.\n", "--report");
+    printf("Available Commands:\n");
+    printf(" %-15s get size in 512-byte sectors.\n", "--getbsz");
+}
+
+static inline struct block_dev* chk_dev_avail(char* dev_name) {
+	struct block_dev *bdev;
+
+	bdev = block_dev_find(dev_name);
+	if (!bdev) {
+		printf("Block device \"%s\" not found.\n", dev_name);
+		return 0;
+	}
+
+	return bdev;
+}
+
+int main(int argc, char **argv) {
+    struct block_dev **bdevs, *bdev;
```

Comment:
Please use tabs in start of line as described in Embox-Code-Style ([https://github.com/embox/embox/wiki/Code-Style](https://github.com/embox/embox/wiki/Code-Style))